.CI/create_feedstocks.py

```diff
@@ -253,12 +253,9 @@ def write_token(name, token):
             )
 
     # Update status based on remote
-    subprocess.check_call(['git', 'stash', '--keep-index'])
     subprocess.check_call(['git', 'stash'])
     subprocess.check_call(['git', 'pull', '--rebase'])
-    subprocess.check_call(['git', 'stash', 'pop'])
-    subprocess.check_call(['git', 'add', '.'])
-    subprocess.check_call(['git', 'stash', 'pop'])
+    subprocess.check_call(['git', 'stash', 'pop', '--index'])
```

Comment:
Sometimes this fails due to conflicts. So had to adjust to PR ( https://github.com/conda-forge/staged-recipes/pull/1473 ).